r/R/dataset-write.R

```diff
@@ -44,7 +44,8 @@
 #' @param filesystem A [FileSystem] where the dataset should be written if it is a
 #' string file path; default is the local file system
 #' @param ... additional format-specific arguments. For available Parquet
-#' options, see [write_parquet()].
+#' options, see [write_parquet()]. For available Feather options, see
+#' [RecordBatchFileWriter$create()].
```

Comment:
I don't think that doc link will work, and if you were to do `make doc`, this will probably fail check due to a bad xref. The options also aren't well documented there, nor are they all exposed in write_feather, so it might be best to enumerate them here.

Reply:
Will do